arch/x86/kernel/cpu/resctrl/rdtgroup.c

```diff
@@ -142,14 +142,14 @@ static int closid_alloc(void)
 	if (closid == 0)
 		return -ENOSPC;
 	closid--;
-	closid_free_map &= ~(1 << closid);
```

Comment:
this patch is missing lockdep_assert_held in this function (https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=5d920b6881f2249be3a028ce0a7f31c5cc61b1ee)

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev